whats-new-panel.yaml

```diff
@@ -135,3 +95,40 @@
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_PROTECTIONS_DASHBOARD_78
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_PROTECTIONS_DASHBOARD_78
+    published_date: 1593475200000
+    title:
+      string_id: cfr-whatsnew-protections-header
+    icon_url: chrome://browser/content/logos/tracking-protection.svg
+    icon_alt:
+      string_id: cfr-whatsnew-protections-icon-alt
+    body:
+      string_id: cfr-whatsnew-protections-body
+    cta_url: protections
+    cta_type: OPEN_ABOUT_PAGE
+    link_text:
+      string_id: cfr-whatsnew-protections-cta-link
+  targeting: firefoxVersion >= 78
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_BETTER_PDF_78
+  template: whatsnew_panel_message
+  order: 2
+  content:
+    bucket_id: WHATS_NEW_BETTER_PDF_78
+    published_date: 1593475200000
+    title:
+      string_id: cfr-whatsnew-better-pdf-header
+    body:
+      string_id: cfr-whatsnew-better-pdf-body
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/pdf-viewer
+    cta_type: OPEN_URL
+    link_text:
+      string_id: cfr-whatsnew-protections-cta-link
```

Comment:
This points to the protection cta link, is that intended?

Reply:
No, thanks for catching it.